src/api/app/components/notification_filter_component.rb

```diff
@@ -0,0 +1,40 @@
+class NotificationFilterComponent < ApplicationComponent
```

Comment:
This looks like a rebase problem. This class is gone in master...

Reply:
Exactly. Removed already